internal/api/api_test.go

```diff
@@ -0,0 +1,53 @@
+package api
```

Comment:
Could we rename this to `base_test.go` after the file it's testing?

Reply:
Done.